<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pastebinConfig</class>
 <widget class="QWidget" name="pastebinConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>96</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pastebin Config Dialog</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textServerLabel">
     <property name="text">
      <string>Pastebin server:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="KComboBox" name="textServer">
     <item>
      <property name="text">
       <string>Pastebin.ca</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pastebin.com</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="imageServerLabel">
     <property name="text">
      <string>Imagebin server:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="KComboBox" name="imageServer">
     <item>
      <property name="text">
       <string>Imagebin.ca</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Imageshack.us</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="historySizeLabel">
     <property name="text">
      <string>History size:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="KIntSpinBox" name="historySize">
     <property name="value">
      <number>3</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
